sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConverter.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace Azure.Messaging.ServiceBus.Amqp
 {
-    internal static class AmqpMessageConverter
+    internal class AmqpMessageConverter
```

Comment:
I needed to make this class non-static in order to mock it in test cases. If another approach is preferred here, I am open to suggestions.